### PR TITLE
[nonmodular]lets condoms be put into wallets

### PR DESCRIPTION
**Pull request by thestubborn**
## About The Pull Request

title
## How This Contributes To The Skyrat Roleplay Experience

there are three things in every wallet
some special coin you got as change one time
dust
and a condom that's probably older than the civil rights movement
now the third can fit in, for IMMERSION

## Changelog

:cl:
expansion: condoms have been shrunk to fit wallets, some how
/:cl:

### CODE REVIEW

**Comment by GoldenAlpharex** [comment]
I'm not sure anyone sane would put an used condom back in their wallet.

**Comment by tf-4** [comment]
It's bad for the health of the condom to put it in your wallet. This is encouraging unsafe sex. 👎 from me.

**Comment by thestubborn** [comment]
> I'm not sure anyone sane would put an used condom back in their wallet.

i don't think anyone sane would work at this station
and, it works on the unopened pack

**Comment by thestubborn** [comment]
> It's bad for the health of the condom to put it in your wallet. This is encouraging unsafe sex. 👎 from me.

I'll have unsafe sex with you

**Comment by Gandalf2k15** [comment]
What is sex? We don't do that here.

**Comment by thestubborn** [comment]
lies and slander

**Comment by thestubborn** [comment]
> I'm not sure anyone sane would put an used condom back in their wallet.

(but seriously if it'll get this merged I'll make it the unopened pack)

File: code/game/objects/items/storage/wallets.dm [hunk 1]
```diff
@@ -39,6 +39,10 @@
 		/obj/item/reagent_containers/syringe,
 		/obj/item/reagent_containers/pill,
 		/obj/item/screwdriver,
+		// SKYRAT EDIT BEGIN
+		/obj/item/condom_pack,
+		/obj/item/clothing/sextoy/condom,
```
Comment (thestubborn) [inline code/game/objects/items/storage/wallets.dm]:
```suggestion
```

**Comment by thestubborn** [comment]
@GoldenAlpharex ok fair, yeah

**Review by GoldenAlpharex — APPROVED** [review]
Sure I guess.

**Pull request merged**